javascript-loops/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+getNumbersToTen();
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+getEvenNumbersToTwenty();
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+repeatWord();
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string);
```

Comment:
You need to log "each" character like: `j`, `a`, `v`, `a` and so on.

Reply:
Forgot to `[i]`(